domains/luoduyu.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "luoduyu",
+        "email": "luoduyu@hotmail.coom"
```

Might be a type(.coom)

```suggestion
        "email": "luoduyu@hotmail.com"
```